stix_shifter_modules/sentinelone/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,645 @@
+    "key": "process.extensions.x-sentinelone-process.md5_image",
+    "object": "process"
+  },
+  "tgtProcImageMd5": {
+    "key": "process.extensions.x-sentinelone-process.md5_image",
+    "object": "process"
+  },
+  "srcProcImageSha256": {
+    "key": "process.extensions.x-sentinelone-process.sha256_image",
+    "object": "process"
+  },
+  "tgtProcImageSha256": {
+    "key": "process.extensions.x-sentinelone-process.sha256_image",
+    "object": "process"
+  },
+  "srcProcSubsystem": {
+    "key": "process.extensions.x-sentinelone-process.sub_system",
+    "object": "process"
+  },
+  "tgtProcSubsystem": {
+    "key": "process.extensions.x-sentinelone-process.sub_system",
+    "object": "process"
+  },
+  "srcProcSessionId": {
+    "key": "process.extensions.x-sentinelone-process.session_id",
+    "object": "process"
+  },
+  "tgtProcSessionId": {
+    "key": "process.extensions.x-sentinelone-process.session_id",
+    "object": "process"
+  },
+  "srcProcActiveContentType": {
+    "key": "process.extensions.x-sentinelone-process.active_content_type",
+    "object": "process"
+  },
+  "tgtProcActiveContentType": {
+    "key": "process.extensions.x-sentinelone-process.active_content_type",
+    "object": "process"
+  },
+  "srcProcActiveContentFileId": {
+    "key": "process.extensions.x-sentinelone-process.active_content_field",
+    "object": "process"
+  },
+  "tgtProcActiveContentFileId": {
+    "key": "process.extensions.x-sentinelone-process.active_content_field",
+    "object": "process"
+  },
+  "srcProcActiveContentPath": {
+    "key": "process.extensions.x-sentinelone-process.active_content_path",
+    "object": "process"
+  },
+  "tgtProcActiveContentPath": {
+    "key": "process.extensions.x-sentinelone-process.active_content_path",
+    "object": "process"
+  },
+  "srcProcActiveContentHash": {
+    "key": "process.extensions.x-sentinelone-process.active_content_hash",
+    "object": "process"
+  },
+  "tgtProcActiveContentHash": {
+    "key": "process.extensions.x-sentinelone-process.active_content_hash",
+    "object": "process"
+  },
+  "srcProcActiveContentSignedStatus": {
+    "key": "process.extensions.x-sentinelone-process.active_content_signed_status",
+    "object": "process"
+  },
+  "tgtProcActiveContentSignedStatus": {
+    "key": "process.extensions.x-sentinelone-process.active_content_signed_status",
+    "object": "process"
+  },
+  "srcProcRpid": {
+    "key": "process.extensions.x-sentinelone-process.rpid",
+    "object": "process"
+  },
+  "srcProcTid": {
+    "key": "process.extensions.x-sentinelone-process.tid",
+    "object": "process"
+  },
+  "srcProcParentStartTime": {
+    "key": "process.extensions.x-sentinelone-process.parent_process_start_time",
+    "object": "process"
+  },
+  "srcProcParentImagePath": {
+    "key": "process.extensions.x-sentinelone-process.parent_image_path",
+    "object": "process"
+  },
+  "srcProcParentUid": {
+    "key": "process.extensions.x-sentinelone-process.parent_uid",
```

Comment:
Wouldn't this be the `pid` on the parent process?

Reply:
SrcProcParentUid contains string data hence can not map to pid (pid is a integer).